test/greenbar/renderers/hipchat_processor_test.exs

```diff
@@ -169,4 +169,54 @@ defmodule Greenbar.Test.Test.HipChatRendererTest do
     assert expected == rendered
   end
 
+
+  test "omits a newline after attachments that are each a single line of output" do
+    directives = [%{"name" => "attachment",
+                    "fields" => [],
+                    "children" => [%{"name" => "text", "text" => "Attachment 1"}]},
+                  %{"name" => "attachment",
+                    "fields" => [],
+                    "children" => [%{"name" => "text", "text" => "Attachment 2"}]},
+                  %{"name" => "attachment",
+                    "fields" => [],
+                    "children" => [%{"name" => "text", "text" => "Attachment 3"}]},
+                  %{"name" => "attachment",
+                    "fields" => [],
+                    "children" => [%{"name" => "text", "text" => "Attachment 4"},
+                                   %{"name" => "newline"},
+                                   %{"name" => "text", "text" => "Plus a newline"}]},
+                  %{"name" => "attachment",
+                    "fields" => [],
+                    "children" => [%{"name" => "text", "text" => "Attachment 5"},
+                                   %{"name" => "newline"},
+                                   %{"name" => "text", "text" => "Plus a newline"}]},
+                  %{"name" => "attachment",
+                    "fields" => [],
+                    "children" => [%{"name" => "text", "text" => "Attachment 6"}]},
+                  %{"name" => "text",
+                    "text" => "Random text"},
+                  %{"name" => "newline"},
+                  %{"name" => "attachment",
+                    "fields" => [],
+                    "children" => [%{"name" => "text", "text" => "Attachment 7"}]}]
+
+    expected = """
+    Attachment 1<br/>
+    Attachment 2<br/>
+    Attachment 3<br/>
+    Attachment 4<br/>
+    Plus a newline<br/>
```

Comment:
Hmm... it's not immediately clear from the output that "Plus a newline" is part of Attachment 4. Alternatively, it's not clear that "Random text" below is _not_ part of attachment 6.

What happens if we just don't add a `<br/>`?